### PR TITLE
add multisig tradeoffs note and doc

### LINKED ISSUE #875: Multisig documentation about being able to verify cosigners xpubs
Should we warn the user about using devices that can verify cosigners xpubs

see https://shiftcrypto.ch/blog/how-nearly-all-personal-hardware-wallet-multisig-setups-are-insecure/

### PR DESCRIPTION
**Pull request by djpnewton**
fix #875

any improvements to text welcomed

### CODE REVIEW

**Comment by stepansnigirev** [comment]
Specter-DIY supports all the features in the table.
Latest ColdCard firmware also shows xpubs in slip132 form (Zpub)

**Comment by djpnewton** [comment]
thank you, updated

**Comment by ben-kaufman** [comment]
Thanks @djpnewton, can you please just add Cobo Vault on the table as well. They have all the listed features I believe.

**Comment by djpnewton** [comment]
done

File: src/cryptoadvance/specter/templates/wallet/new_wallet/new_wallet.jinja [hunk 1]
```diff
@@ -32,6 +32,8 @@
 				{% if wallet_type == 'multisig' %}
 					We use sorted multisig (BIP-67), so <b>order is NOT important.</b><br>
 					<br>
+					Be aware of <a style="color: #fff" href="https://github.com/cryptoadvance/specter-desktop/blob/master/docs/multisig-security-tradeoffs.md">multisig security tradeoffs</a>.<br>
```
Comment (ben-kaufman) [inline src/cryptoadvance/specter/templates/wallet/new_wallet/new_wallet.jinja:35]:
```suggestion
					Be aware of <a style="color: #fff" href="https://github.com/cryptoadvance/specter-desktop/blob/master/docs/multisig-security-tradeoffs.md" target="_blank">multisig security tradeoffs</a>.<br>
```

**Comment by ben-kaufman** [comment]
Looks good, thanks, just need to make the link target a new tab.

**Comment by djpnewton** [comment]
updated and rebased

**Review by ben-kaufman — APPROVED** [review]

**Comment by JFHCFC** [comment]
Despite updating to the latest versions, the error persists "Unverified inputs"
I cannot transfer bsv to another wallet

I’ve got:
- Electrumsv 1.3.13
- Ledger Live - 2.29.0
- Firmware on Nano S - 2.0.0
- Installed BTC - 1.6.2
- Installed BCH-  1.6.2

**Pull request merged**